source/_patterns/01-atoms/svgicon/_svgicon.twig

```diff
@@ -0,0 +1,14 @@
+{#
+/**
```

Comment:
Let's not use the js comment header in Twig. It looks nice, but let's just use what's in other twig files for now.

Reply:
New default behavior with generator.